### PR TITLE
add class function 'delete_file()' to S3Bucket class

### PR DESCRIPTION
**Pull request by VictorSundberg**
add a fuction to allow to delete files within s3 storage.

### CODE REVIEW

File: tibber_aws/s3.py [hunk 1]
```diff
@@ -49,6 +49,10 @@ async def load_data(self, key, if_unmodified_since=None):
         data, _, state = await self.load_data_metadata(key, if_unmodified_since)
         return data, state
 
+    async def delete_file(self, key):
+        await self.init_client_if_required()
+        self._client.delete_object(Bucket=self._bucket_name, Key=key)
```
Comment (Danielhiversen) [inline tibber_aws/s3.py:54]: I would assume it should be
```suggestion
        await self._client.delete_object(Bucket=self._bucket_name, Key=key)
```

Comment (VictorSundberg) [reply tibber_aws/s3.py]: Done

**Comment by VictorSundberg** [comment]
BTW, Should I uptick the version in any way?

**Review by Danielhiversen — APPROVED** [review]
👍 

You can make a new release afterwards https://github.com/tibbercom/tibber-pyAws#create-release

**Pull request merged**